lib/dependencies/DepBlockHelpers.js

```diff
@@ -33,7 +33,7 @@ DepBlockHelpers.getDepBlockPromise = function(depBlock, outputOptions, requestSh
 				"])";
 		}
 	}
-	return "Promise.resolve()";
+	return "new Promise(function(resolve) { resolve(); })";
```

Comment:
And yes, it is in the spec: http://www.ecma-international.org/ecma-262/6.0/#sec-promise.resolve